src/controllers/problem.ts

```diff
@@ -90,7 +90,7 @@ const problemController = {
             res.status(statusCodes.MISSING_PARAMS).json(errors.formatWith(errorMessage).array()[0]);
         } else {
             try {
-                const { problemId } = req.params;
+                const problemId: string  = req.params.problemId;
```

Comment:
we should keep the deconstructed format ( const { problemId } = req.params;) to stay consistent with the rest of the code across the microservices